Makefile

```diff
@@ -11,6 +11,8 @@ export PROJECT := $(or $(PROJECT),ehpr)
 export NODE_ENV ?= development
 export ENV_NAME ?= dev
 export POSTGRES_USERNAME = freshworks
+export CHES_CLIENT_ID ?= EHPR_SERVICE_CLIENT
+export MAIL_FROM ?= noreply@gov.bc.ca
```

Comment:
`EHPRDoNotReply@gov.bc.ca` is the `from` documented in the EHPR text document, should we use that here instead?

Reply:
I think we probably should